### PR TITLE
add aishell recipe

### PR DESCRIPTION
**Pull request by fanlu**
take example by #57 and other previous pr, make a recipe with aishell mandarin dataset
result is
```
2020-12-28 10:18:09,533 INFO [decode.py:222] %WER 23.62% [15221 / 64428, 1372 ins, 1847 del, 12002 sub ]
2020-12-28 10:18:09,533 INFO [decode.py:226] %CER 14.49% [15180 / 104765, 415 ins, 418 del, 14347 sub ]
```
@danpovey @csukuangfj please have a review

### CODE REVIEW

**Comment by danpovey** [comment]
Great!!
Guys, unless there are bugs, I am inclined to merge this without too much scrutiny.  Snowfall is not about getting things perfect or uniform.

File: egs/aishell/asr/simple_v1/aishell.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
+"""
+About the Aishell corpus
+Aishell is an open-source Chinese Mandarin speech corpus published by Beijing Shell Shell Technology Co.,Ltd.
+publicly availble on https://www.openslr.org/33
+"""
+
```
Comment (csukuangfj) [inline egs/aishell/asr/simple_v1/aishell.py:6]: Would you mind writing the author copyright info here, i.e., your name and your company?

**Comment by csukuangfj** [comment]
+1

File: egs/aishell/asr/simple_v1/aishell.py (hunk 1 above)
Comment (fanlu) [reply egs/aishell/asr/simple_v1/aishell.py:11]: Done

Comment (csukuangfj) [reply egs/aishell/asr/simple_v1/aishell.py:11]: thanks

**Comment by pzelasko** [comment]
Would you consider adding aishell manifest preparation to Lhotse?

**Comment by fanlu** [comment]
> Would you consider adding aishell manifest preparation to Lhotse?
> […](#)
> Wiadomość napisana przez Fangjun Kuang ***@***.***> w dniu 12/28/20, o godz. 05:19: ﻿ Merged #59 into master. — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub, or unsubscribe.

OK， Let me move aishell.py to Lhotse

**Pull request merged**